client-v2/src/components/FrontsEdit/CollectionComponents/Collection.tsx

```diff
@@ -216,14 +216,14 @@ const createMapStateToProps = () => {
         collectionId: id
       }),
       isCollectionLocked: isCollectionLockedSelector(state, id),
```

Comment:
Should this be: 

```suggestion
      isCollectionLocked: selectIsCollectionLocked(state, id),
```

The imports for `hasUnpublishedChangesSelector` and `isCollectionLockedSelector` also need updating in this file.